tsdoc/src/configuration/TSDocConfiguration.ts

```diff
@@ -182,6 +192,33 @@ export class TSDocConfiguration {
     }
   }
 
+  /**
+   * Overwrite the supported HTML elements.
+   */
+  public setSupportedHtmlElements(htmlTags: string[]): void {
+    this._supportedHtmlElements.clear();
+    for (const htmlTag of htmlTags) {
+      this._supportedHtmlElements.add(htmlTag);
+    }
+  }
+
+  /**
+   * Overwrite the validator's existing value for `reportUnsupportedHtmlElements`.
+   */
+  public setReportUnsupportedHtmlElements(reportUnsupportedHtmlElements: boolean): void {
+    this.validation.reportUnsupportedHtmlElements = reportUnsupportedHtmlElements;
+  }
```

Comment:
I think we can delete this API. The `validation` property is a public API and it's not readonly.

Reply:
removed